wiki/Tournaments/COT/2022/en.md

```diff
@@ -0,0 +1,88 @@
+| Commentator | ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164),  ![][flag_RU] [_Insomnia](https://osu.ppy.sh/users/8928855), ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464), ![][flag_RU] [ThankYou](https://osu.ppy.sh/users/4571241) |
+| Referee | ![][flag_RU] [_Insomnia](https://osu.ppy.sh/users/8928855), ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464), ![][flag_RU] [2zz](https://osu.ppy.sh/users/8201267) |
+| Designers | ![][flag_RU] [ThankYou](https://osu.ppy.sh/users/4571241), ![][flag_RU] [2zz](https://osu.ppy.sh/users/8201267) |
+| Spreadsheet manager | ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464) |
+| Wiki editor | ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164) |
+
+## Links
+
+- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/1iie_zXj1GbS_Sx4MyWdGOYD9raofZ46uomMbVW6wm48/edit?usp=sharing)**
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1493650)
+- [Discord server](https://discord.gg/ZqCdWBKGrM)
+- [Challonge brackets](https://challonge.com/ru/cot22)
+- [Livestream channel](https://www.twitch.tv/playosuru/)
+
+## Ruleset
+
+### General rules
+
+1. Map scoring is based on **[ScoreV2](/wiki/Score#scorev2).**
+2. The match lobby will be created by the referee, and the players will be invited.
+3. Players have a strict 10-minute grace period available to play the match. If 10 minutes pass and a player does not show up the match, the other player will receive a win by default.
+4. Warm ups aren't allowed.
+5. During the lobby it is not allowed to show any form of harassment during livestreaming. By organizers after breaking the rule the player may be disqualified.
+6. At the start of the match, both players roll. The roll winner decides to ban or pick first.
+7. The referee is not allowed to play in the match, which means the referee should not be in the lobby during the match process.
+8. Each player is allowed 1 or 2 bans per match. Banned maps are removed from the mappool for the duration of the match and not allowed to be picked. A player is not allowed to ban 2 maps from the same mod pool.
+9. Players have 2 minutes to select a map; if they fail to choose in the allotted time, the map choice will be random.
+10. Each player may use a tactical timeout with the 2 minute length which can be used once during the tournament. The tactical timeout is optional and it is not required to use it.
+11. After choosing a map players have a 2 minute preparation. A referee will start the match when both players are ready. After that, when the timer ends, referee starts the match.
+12. NF is forced on every map.
+13. In the case of a disconnection: if the disconnection happened in the first 30 seconds during the match, it will be replayed. If the disconnection happened after 30 seconds, then the score won't count.
```

Comment:
```suggestion
13. If a player disconnects in the first 30 seconds during the match, it will be replayed. If the disconnect happened after 30 seconds, then the disconnected player's score won't count.
```

Reply:
```suggestion
13. If a player disconnects during the first 30 seconds of the match, it will be replayed. If the disconnect happened after 30 seconds, then the disconnected player's score will not count.
```